src/main/java/frc/robot/RobotContainer.java

```diff
@@ -134,6 +135,7 @@ public RobotContainer() {
                         ? new Traverser(new SingleMotorIoNeo550Brushless(CONSTANTS.getTraverserMotorId(),
                                 CONSTANTS.isTraverserInverted()))
                         : null;
+                climber = CONSTANTS.hasClimberSubsystem() ? new Climber() : null;
```

Comment:
Should this be a simulated climber for odometry replay?

Reply:
You can move it down to the "Initialize Subsystems without Simulation and/or Log Replay Mode" region.